SampleProject/SampleProject/Resources/Localizable.swift

```diff
@@ -0,0 +1,58 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+import Foundation
+
+// swiftlint:disable superfluous_disable_command file_length implicit_return
+
+// MARK: - Strings
+
+// swiftlint:disable explicit_type_interface function_parameter_count identifier_name line_length
+// swiftlint:disable nesting type_body_length type_name vertical_whitespace_opening_braces
+internal enum Localizable {
+  internal enum Favorites {
+    /// %d Yorum %d Beğeni
+    internal static func recipeStats(_ p1: Int, _ p2: Int) -> String {
+      return Localizable.tr("Favorites", "recipeStats", p1, p2)
+    }
+    /// TÜMÜNÜ GÖR
+    internal static let seeAllButtonTitle = Localizable.tr("Favorites", "seeAllButtonTitle")
+    /// %d Tarif %d Takipçi
+    internal static func userStats(_ p1: Int, _ p2: Int) -> String {
+      return Localizable.tr("Favorites", "userStats", p1, p2)
+    }
+  }
+  internal enum Recipes {
```

Comment:
need to explicit internal? I'm not sure your requirements right here but it is looking unnecessary.

Reply:
This file is automatically generated by Swiftgen. We do not enforce any rules (swiftlint or style guide) for this file.